packages/auto-catalog/src/client/components/AutoCatalog.ts

```diff
@@ -249,7 +260,7 @@ export default defineComponent({
                 ),
                 children.length
                   ? h(
-                      "ul",
+                      props.index ? "ol" : "ul",
```

Comment:
Are nesting ol markers be `1.1` `1.1.1` in all kinds of devices? I think not, the styles needs to be handled correctly.